docs/notebooks-b.md

```diff
@@ -0,0 +1,42 @@
+# Notebooks
+
+Jupyter notebooks sit at the core of machine learning expeditions at Substratus, the driver seat on your journey of refining raw data into a finely-tuned, deployed model.
+
+## Key Features
+
+**1. Emulated Environments.**
+
+Imagine meeting your doppelganger, a perfect replica of yourself who thinks, acts, and even makes the same errors just as you would. Those are exactly the traits Substratus notebooks possess. They replicate the ML environments where your tasks come alive using the consistency provided by containers.
```

Comment:
seems to use too many words to get the poitn across

Reply:
+1 and I don't really think the analogy works either